lib/tlitest/test_tlog.py

```diff
@@ -2,48 +2,56 @@
 
 import os
 import ast
+import sys
 import stat
 import time
-import pytest
-import pexpect
+import socket
 import inspect
-from pexpect import pxssh
-from systemd import journal
+from shutil import copyfile
 from tempfile import mkdtemp
+from systemd import journal
+
+import pexpect
+from pexpect import pxssh
 
+import pytest
 
 def journal_find_last():
+    """ helper to get the last journal entry """
     j = journal.Reader()
     j.seek_tail()
     entry = j.get_previous()
-    while('TLOG_REC' not in entry):
+    while 'TLOG_REC' not in entry:
         entry = j.get_previous()
     return entry
```

Comment:
This will return an entry, even if there are no tlog-rec entries in the journal and that might get confusing. I would return `None` if an entry is not found.

Reply:
Actually in my testing a little while ago it looped indefinitely which is probably worse.

I added another commit to modify to code to add a conditional for len(entry) != 0 to while and raise an exception if TLOG_REC not found.  There may be cases later where we might use this to search for no tlog-rec entries in a journal but, right now this is only used for cases where they exist.  So, if they don't exist, probably best to raise an exception to fail the test case?